etcdserver/etcdserverpb/rpc.proto

```diff
@@ -469,6 +469,11 @@ message RoleRevokeRequest {
 
 message AuthEnableResponse {
   ResponseHeader header = 1;
+
+  enum Result {
```

Comment:
i think you can return error in gRPC. we should not put error string in response.